review-notifications/src/Github.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import axios from 'axios';
+
+class Github extends Component {
+	constructor(props) {
```

Comment:
This formatting is off. Consider adding prettier (https://prettier.io/) which will format your code correctly and consistently so things like those don't get forgotten :)